### ISSUE TITLE
A11y swarm for 'Features' and 'Top Stories' components

### ISSUE THREAD

**Issue by lizcameron** [bbc/simorgh#6088]
**Is your feature request related to a problem? Please describe.**
Carry out a dev a11y swarm for the 'Features and analysis' and Top stories' onward journey components once they are both completed.

Screenreader UX and a11y criteria can be found in Jira (ask @lizcameron for a link if you don't have one) - Zeplin design links are also in Jira:
Features and Analysis: NEWSART-914
Top Stories: NEWSART-913

**Describe the solution you'd like**
Go through the accessibility checklist and note any issues. These will then be prioritised and new issues can be made to address them.

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**
-  Some a11y issues had already been noticed during development, see this PR https://github.com/bbc/simorgh/pull/5751#issuecomment-598268665 and https://github.com/bbc/simorgh/pull/5751#issuecomment-599515097 these will need to be captured as part of this a11y swarm and prioritised for fixing

**Comment by HarveyPeachey** [comment]
**Dev/A11y Swarm**

-https://bbc.github.io/accessibility-news-and-you/accessibility-news-and-testers
  - [x] Axe
  - [x] Design
  - [x] Colour Contrast
  - [x] Colour blindness
  - [x] Windows HCM
  - [x] FF Colour Preferences
  - [x] Text 200%, Page 200%
 Cannot see the components: issue to resolve problem: https://github.com/bbc/simorgh/issues/6485
  - [x] Icons
  - [x] Heading Order
  - [x] Check accesibility of browser DOM
  - [x] Keyboard Navigation 

**_Screen Readers (IE11) - Harvey:_** 
- [x] ZoomText Magnifier/Reader
- [x] Dragon NaturallySpeaking
- [x] JAWS
- [x] Read&Write
- [x] NVDA (Firefox)


**_Mac, iOS, and Android:_** 
- [x] VoiceOver iOS (iPhone/iPad)
- [x] VoiceOver OS (Mac)
- [X] TalkBack (Chrome on Android)
 7. Landmarks selected: "Features & Analysis" is read out then it reads out "region Features and Analysis heading 2 list"

**Comment by HarveyPeachey** [comment]
A summary is still present for Features and Analysis, which only shows on the tablet breakpoint. This means the text is still visually hidden for other breakpoints, which causes them to be read out by ~Jaws~ Read & Write. @greenc05 From my understanding it was decided the summary would be left how it is, is this still the case?

**Comment by lizcameron** [comment]
Thanks @HarveyPeachey  -  we have a bug raised in the backlog for this issue, as it affects multiple page types using the Story Promo component.

**Comment by greenc05** [comment]
@HarveyPeachey yeah the summary text is display none and will be read with Read&Write, we can't fix that, just the way that tool works at the moment... @lizcameron please can you close the issue for this particular bug in the backlog if there is one?

**Comment by Simon2828** [comment]
When running axe in the full page a11y swarm, I saw I missed an issue in the Features & Analysis component:
_ARIA roles used must conform to valid values. Element location:_ `a[href="\/gahuza\/amakuru-23029740"] > span[role="text"]`. 
The suggested fix is: _Role must be one of the valid ARIA roles: text_

**Comment by greenc05** [comment]
Thanks @Simon2828 we have an issue with axe-core to address this https://github.com/dequelabs/axe-core/issues/1597

**Comment by HarveyPeachey** [comment]
Closing this issue as swarm has been completed
